cluster-scope/overlays/edu/kustomization.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  nerc.mghpcc.org/kustomized: "true"
+
+resources:
+- ../common
+- ../../bundles/acct-mgt
+- ../../bundles/patch-operator
+- ../../bundles/solr-helm-repo
+- ../../bundles/ansible-automation-platform-operator
+- ../../bundles/openshift-serverless-operator
+- ../../bundles/authorino-operator
+- ../../bundles/servicemesh-operator
+- ../../bundles/amq-broker-rhel8-operator
+- ../../bundles/crunchy-postgres-operator
+- ../../bundles/amq-streams-operator
+- ../../bundles/openshift-pipelines-operator
+- ../../bundles/openshift-opentelemetry-operator
+- ../../bundles/rhods-operator
+- ../../bundles/node-feature-discovery
+- ../../bundles/nvidia-gpu-operator
+- ../../bundles/openshift-custom-metrics-autoscaler-operator
+- ../../bundles/clusterissuer-http01
+- ../../bundles/koku-metrics-operator
+- ../../bundles/autopilot
+- ../../bundles/mongodb-operator
+- ../../bundles/nagios-monitoring
```

Comment:
@jtriley can we add the [xdmod-reader bundle](https://github.com/OCP-on-NERC/nerc-ocp-config/tree/main/cluster-scope/bundles/xdmod-reader)? it'll create a serviceaccount and a token that I can use for gathering metrics from the cluster. Thanks!

Reply:
@naved001 Let's rename it instead - what would be a representative name?

Reply:
`moc-billing` or `moc-invoicing` or `moc-metrics-collector`. Either of those is fine.

Reply:
OK let's go with `moc-metrics-collector`. I'm going to merge this and do that in a separate PR after I finish the coldfront portion of today's maintenance.